gatsby-browser.js

```diff
@@ -1,5 +1,9 @@
+const SwaggerUI = require('swagger-ui')
```

Comment:
Does this mean that Swagger runs on every page? If do, is there a way to only load it on the pages we'll need it for?

Reply:
Not sure, I think @superbiche might know?

Reply:
Wow, I missed that. I'm going to do a test and propose a change to just load swagger on that one page.

Reply:
OK I just pushed a commit that:

- Uses `onCreateWebpackConfig` to say "Gatsby, when you're building the site with SSR, and therefore there is no `window` object, just chill and don't bother about Swagger."
- Loads swagger UI in the `SwaggerSandbox` component.